qemu/tests/cfg/win_virtio_serial_data_transfer.cfg

```diff
@@ -0,0 +1,24 @@
+- win_virtio_serial_data_transfer:
+    type = win_virtio_serial_data_transfer
+    only Windows
+    virtio_ports = "com.redhat.rhevm.vdsm1 com.redhat.rhevm.vdsm2"
+    virtio_port_type = "serialport"
+    guest_scripts = VirtIoChannel_guest_receive.py;windows_support.py
+    check_vioser_status_cmd = verifier /querysettings
+    vioser_verify_cmd = verifier.exe /standard /driver vioser.sys
+    host_send_script = serial-host-send.py
+    guest_receive_script = VirtIoChannel_guest_receive.py
+    repeat_times = 1000
+    variants:
+        - interrupt:
+             interrupt = yes
+             variants:
+                 - with_reboot:
+                     sub_test = reboot
```

Comment:
you get with "sub_step"in code, but set "sub_test" here
sub_step = params.get("sub_step", "reboot")